### ISSUE TITLE
Mission Control, Image Capture, Automator.

### ISSUE THREAD

**Issue by tvalkx** [tinalatif/flat.icns#212]
Hello,

I commend you on your streamlined graphics skills. 
Thank you for making this available to the public.
Please make the following icons;

Mission Control
Image Capture
Automator

Kind Regards.

**Comment by wjramos** [comment]
I could be remembering incorrectly but I thought there were icons in the set for image capture and mission control. I'd have to check again - either way we need one for automator

**Comment by wjramos** [comment]
![missioncontrol](https://cloud.githubusercontent.com/assets/5625810/8816644/92809ea4-2fde-11e5-942e-046ed0e6914f.png)

**Comment by wjramos** [comment]
Automator has been one I've been contemplating for a while. I really want to capture the essence - but I don't know if that is the cheesy robot, the pipe he is holding, or what. if it was the robot, do we need to show the whole robot?

**Comment by wjramos** [comment]
![automator](https://cloud.githubusercontent.com/assets/5625810/8816770/fd9fc25e-2fdf-11e5-96b1-2619798b77a0.png)

**Comment by tinalatif** [comment]
I agree that Automator is a weird one. I'm usually strongly in favour for sticking pretty closely to the original icon, but in this case, I'm happy just going with something that somewhat resembles the original. I'd be happy with the version you made

(side note, I also totally thought there were already Mission Control and Image Capture icons, ha)

**Comment by wjramos** [comment]
I realized I've been using Page Layers as my image capture icon
![page layers](https://cloud.githubusercontent.com/assets/5625810/8818651/ed155c96-2ff8-11e5-908f-8643085a62de.png)
